AnnService/GPUSSDServing.vcxproj

```diff
@@ -113,7 +118,7 @@
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
-    <ClCompile Include="src\SSDServing\main.cpp" />
+    <!--<ClCompile Include="src\SSDServing\main.cpp" />-->
```

Comment:
remove this itemgroup?

Reply:
Oh, I actually should put it back, since we have already fixed the Build issue by having TruthSet.cpp separately.